phone_verify/backends/kavenegar.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+
+# Third Party Stuff
+from kavenegar import KavenegarAPI, APIException, HTTPException
+
+# Local
+from .base import BaseBackend
+
+
+class KavenegarBackend(BaseBackend):
+    def __init__(self, **options):
+        super(KavenegarBackend, self).__init__(**options)
```

Comment:
```suggestion
        super().__init__(**options)
```

I meant this line :)